### DISCUSSION TITLE
Custom javascript

### DISCUSSION THREAD

**Discussion by scottmando2000** [Paymenter/Paymenter#217] [Ideas]
Add the ability to add custom javascript into the head of every page.

This will be useful to add analytics scripts.

**Comment by finnjosten** [comment]
Security issue for js injections pls don't allow this. Then just add some default js and let users put in their google analytics keys for example

**Reply by 09han09** [reply]
Was about to reply something like that.

**Reply by scottmando2000** [reply]
If instead of just Google Analytics, we could also have the ability to add other analytics.

I use plausible, which is in the following format.
`<script defer data-domain="your.domain" src="https://analytics.your.domain/js/script.js"></script>`

**Reply by finnjosten** [reply]
Sure but then just bake it in paymenter and let a user set the variables dont allow a user to input custom js

**Comment by 09han09** [comment]
You can add your own javascript in the theme files, please checkout https://paymenter.org/docs
